hr_employee_transfer/models/employee_transfer.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+from datetime import date
+from odoo import models, fields, api, _
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
+from odoo.exceptions import Warning
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'Warning'](https://app.codacy.com/manual/numigi/odoo-hr-addons/pullRequest?prid=5137145)